config/jest/setupAfterEnv.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
We can include these definitions in config/jest/index.js` to avoid an additional configuration file